JustSaying.IntegrationTests/AwsTools/WhenCreatingTopicByName.cs

```diff
@@ -26,9 +26,9 @@ protected override SnsTopicByName CreateSystemUnderTest()
             return CreatedTopic;
         }
 
-        public override void PostAssertTeardown()
+        protected override void PostAssertTeardown()
```

Comment:
I see further down the diff there's an `async` version of this base class - possible to use that so this doesn't need the `Wait()`?

Reply:
There is no async teardown mechanism unfortunately.